app/presenters/file_presenter.rb

```diff
@@ -1,9 +1,4 @@
 # app/presenters/file_presenter.rb
 class FilePresenter < Sufia::GenericFilePresenter
-  self.terms = [:dc_type, :title, :creator, :contributor, :description,
-                :tag, :rights, :publisher, :date, :subject, :language,
-                :spatial, :temporal, :abstract, :tableOfContents, :format, 
-		:bibliographicCitation, :provenance, :isReferencedBy, :relation,
-		:rights, :title, :isCitedBy, :isIdenticalTo, :isPartOf,
-		:isVersionOf, :doi, :hdl, :orcid, :rid]
+  self.terms = ScholarsArchiveSchema.properties.map(&:name)
```

Comment:
Is this right? I thought it was self.terms +=, since GenericFile had some terms.